Automattic/jetpack
# Search: Do not redirect to the same page in wp-admin

Fixes pbfA90-tz-p2

#### Changes proposed in this Pull Request:

* Do not redirect when deactivating Search.

#### Jetpack product discussion



Debug notes: p1648143618433359-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Before this patch, spin up a JN running JP master. Don't need to connect even, but activate Jetpack and attempt to go to Appereance->Menus.
* Expected: It goes to the nav-menu.php page.
* Before this patch, it reloads the page you're on.
